### PR TITLE
Added entry for HTML`<output>` tag

### LINKED ISSUE #653: [Entry] HTML: `<output>` tag
### Reason/inspiration (optional)

Added the result of an action done by the user or calculation to the page.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Antonio-Riccelli**
### Description

Closes #653 

### Type of Change

- Adding a new entry

Hi all, 
This is the first draft of the entry. I have run the format/lint scripts before pushing.

Please let me know all your suggestions and corrections. 

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by ethan-master-coding — CHANGES_REQUESTED** [review]
@Antonio-Riccelli On lines 43-46 we should have all the attributes on one line because the other attributes are on 1 line for the other tags.

**Review by ethan-master-coding — CHANGES_REQUESTED** [review]
Forgot 1 thing. We like when there is a screenshot of the output of the example currently in the file.

**Comment by Antonio-Riccelli** [comment]
@ethan-master-coding thanks for your feedback 👍I'm about to push the update.

It was all on one line initially, but I think the formatting/linting scripts changed that. Currently, it puts the closing tag on a new line, I'm not sure why. See below: 
demo3


As we want to add a picture, I thought of making the example a bit more descriptive, as the current one only showed a slider and a number. I added a label showing the actual value of the slider. This way it highlights the difference with the output.

A static image didn't really convey the example well, so I thought of making a gif. 

Let me know if this works, or else I can switch to a different and simpler example.

**Review by ethan-master-coding — APPROVED** [review]
@Antonio-Riccelli  Looks good. We may be able to use a `prettier ignore` or something similar so the checks still pass. A maintainer can help with that since I don't know it too well.

**Review by SSwiniarski — APPROVED** [review]
@Antonio-Riccelli, I did a first review, and I don't have any changes to suggest. On to review number 2.

**Comment by Antonio-Riccelli** [comment]
Hi @SSwiniarski,
I was just wondering if there's anything else I should change on this to allow it to progress?

**Comment by SSwiniarski** [comment]
We're just waiting on a second review.

File: content/html/concepts/tags/terms/output/output.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/output/output.md:17]:
```suggestion
The `<output>` tag displays the result of a calculation or user action. It is commonly used inside of a [`<form>`](https://www.codecademy.com/resources/docs/html/tags/form) element to dynamically show the result of user input. However, the `<output>` tag can be used on its own.
```

File: content/html/concepts/tags/terms/output/output.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/output/output.md:3]:
```suggestion
Description: 'Displays the result of a calculation or user action.'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@Antonio-Riccelli Just had a few comments/suggestion for you but this looks great, otherwise! 😄

**Comment by Antonio-Riccelli** [comment]
Thank you, @Dusch4593! Suggestions applied.

**Review by Dusch4593 — APPROVED** [review]
@Antonio-Riccelli LGTM! 👍🏻

**Comment by Dusch4593** [comment]
Congrats on another contribution to Docs, @Antonio-Riccelli 🙏🏻 🎊 🎉 When the entry goes live, it can be found with this link: https://www.codecademy.com/resources/docs/html/tags/output

**Pull request merged**
